clang/test/3C/typedefs.c

```diff
@@ -0,0 +1,49 @@
+// RUN: 3c -alltypes -addcr %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_ALL","CHECK" %s
+// RUN: 3c -addcr %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_NOALL","CHECK" %s
+// RUN: 3c --addcr --alltypes %s -- | %clang_cc1  -fcheckedc-extension -x c -
+// RUN: 3c --addcr %s -- | %clang_cc1  -fcheckedc-extension -x c -
+//
+// 3c -alltypes -output-postfix=checked %s
+// 3c -alltypes %S/typedefs.checked.c -- | diff %S/typedefs.checked.c -
+// rm %S/typedefs.checked.c
+
+typedef int* intptr;
+//CHECK: typedef _Ptr<int> intptr;
```

Comment:
It occurs to me: What happens if we have a source file with checked types already in it? E.g., what if we had
```
typedef int *intptr;
typedef _Ptr<intptr> PP;
...
```
Would this create any problem for rewriting the `intptr` to be `_Ptr<int>` later? Could add a case like that to this test.

Reply:
works, and added as a test